internal/datastore/crdb/crdb_test.go

```diff
@@ -138,7 +139,6 @@ func TestCRDBDatastoreWithFollowerReads(t *testing.T) {
 	for _, quantization := range quantizationDurations {
 		t.Run(fmt.Sprintf("Quantization%s", quantization), func(t *testing.T) {
 			t.Parallel()
-			require := require.New(t)
```

Comment:
I had to do this in a couple of places because I wanted to use `require.EventuallyWithT`, which passes a `c` object to its child `require.*` calls, but if we're using an aliased `require` we can't do that.